.github/workflows/lint_metadata.yml

```diff
@@ -0,0 +1,37 @@
+name: Spellcheck + Commitlint
```

Comment:
let's call all of these linters.yml and "Linters" name.

Reply:
done